kubernetes/charts-linera/values-dev-gcp.yaml

```diff
@@ -0,0 +1,3 @@
+# Values for charts charts-linera for the dev validator on GCP.
+
+lineraImage: us-docker.pkg.dev/linera-io-dev/linera-docker-repo/zefchain-gcp-test:0.1
```

Comment:
I need to rebuild this package to remove the `zefchain` from the name

Reply:
Yup. Not sure why this was named `zefchain-gcp-test` in the first place.